tests/components/plugwise/test_binary_sensor.py

```diff
@@ -1,6 +1,6 @@
 """Tests for the Plugwise binary_sensor integration."""
 
-from homeassistant.config_entries import ENTRY_STATE_LOADED
```

Comment:
This has changed in HA 2021.6. so this code won't run properly on previous HA versions.
Not sure if this will ever happen for tests. If not, just ignore my comment.

Reply:
Yes, I've noticed this too. Have already tried to update to what I thought was the correct code but it seems not to work.

Reply:
Looks like my update works in your PR :)